Add Mappers' Report: May news post (#5720)
ppy/osu-wiki: news/2021-06-20-mappers-report-may.md

---
layout: post
title: "The Mappers' Report: May"
date: 2021-06-20 20:00:00 +0000
---

Take everything in with the May issue of the Mappers' Report and begin your summer up-to-date with the mapping and modding community!

![](/wiki/shared/news/banners/the-mappers-report.jpg)

This month's issue of the Mappers' Report contains the following topics:

- **[Ranking Criteria Changes](#ranking-criteria-changes)**
- **[Within the Beatmap Nominators and NAT](#within-the-beatmap-nominators-and-nat)**
- **[Mappers' Showcase](#mappers-showcase)**
- **[Mapping Contests](#mapping-contests)**
- **[Around the Community](#around-the-community)**
- **[Conclusion](#conclusion)**

## Ranking Criteria Changes

> Unfortunately, we don't have any Ranking Criteria changes for you this month. But fear not, we have several proposals at hand waiting for further polishing. Be one of the first and check them out right away!

Reviewer: ```suggestion
Unfortunately, we don't have any Ranking Criteria changes for you this month. But fear not, because we have several proposals at hand waiting for further polishing. Be one of the first and check them out right away!
```

Recently breaching through the ranked section, he finally propels ahead with his rendition of [Asriel - Kegare Naki Yume](https://osu.ppy.sh/beatmapsets/710899) which features a simple yet captivating style with its pleasing aesthetic and coherent representation of the music. In similar fashion, his take on [sokoninaru - Tenohira de odoru](https://osu.ppy.sh/beatmapsets/1200657) presents clean-looking streams and apt spacing usage to emphasize the music accordingly.

Other than being a hard rock mapper himself, KuroRev has also been endeavouring to push his first full solo set with the lovely future bass song [YUC'e - Cinderella Syndrome](https://osu.ppy.sh/beatmapsets/1467712), showcasing his mastery on movement and immaculate pattern structure. If you do like his maps, consider clicking that bell on his profile to not miss out on his later bangers. We are eager to see his motivation grow further and hopefully see him create more interesting maps!

## Mapping Contests

> Another month of community-hosted contests is upon us, this time with something sweet for all osu!taiko and osu!catch mappers with some really great prizes. Don't know how to map either mode? That doesn't matter — you should try your luck anyway! Who knows, you may discover some hidden master mapper talent hiding within you.

Reviewer: ```suggestion
Another month of community-hosted contests is upon us, this time with something sweet for all osu!taiko and osu!catch mappers with some really great prizes. Don't know how to map in either mode? That doesn't matter — you should try your luck anyway! Who knows, you may discover some hidden master mapper talent hiding within you.
```

### osu!flushed Taiko Mapping Contest

The [osu!flushed Taiko Mapping Contest](https://osu.ppy.sh/community/forums/topics/1342681) is a brand new contest where the responsibility of choosing the song falls onto your hands. That's right, you will have an *almost* complete freedom in song choice, where the only restriction is that it needs to follow a certain theme.

As for the prizes, the winner gets 6 months of osu!supporter, while the runners-ups of 2nd and 3rd place receive 4 months of osu!supporter.

> Don't wait for the drums to roll - go ahead and sign yourself up via the thread above to show who places the blues and reds the best!

Reviewer: ```suggestion
Don't wait for the drums to roll — go ahead and sign yourself up via the thread above to show who places the blues and reds the best!
```

### osu!catch 2021 Summer Featured Artist Cup

As the summer cracks the door open, the fruits start ripening. To fill your cup with the cream of the crop, be sure to join the [osu!catch Featured Artist Cup](https://osu.ppy.sh/community/forums/topics/1327798) right away!

Featuring songs from our [Featured Artists](https://osu.ppy.sh/beatmaps/artists) who joined the lineup between March 20th - June 20th, this contest will test your mapping capabilities in two difficulties: one being Platter or lower, and the other being Rain or higher.

> But that's not all - this one has some hefty prizes too! 

Reviewer: ```suggestion
But that's not all — this one has some hefty prizes too! 
```

The winner will be rewarded with 100 USD in cash, plus having their set modded by the host and the judges! Should you not make it to the top, the runners-up will still get 20 USD in cash in addition to 4 months of osu!supporter for 2nd place, and the 3rd place will enjoy 2 months of osu!supporter.

> Have we got you interested in getting your hands dirty with the harvest? Then be quick - the sign-ups are closing soon on **June 20th**!

Reviewer: ```suggestion
Have we got you interested in getting your hands dirty with the harvest? Then be quick — the sign-ups are closing soon on **June 20th**!
```

Author: That sentences got completely reworded as the deadline sadly passed

### osu!catch Mapping World Cup 2021

Brought to you by the osu!catch mapping grand-master [Ascendance](https://osu.ppy.sh/users/2931883) for the third time, the [osu!catch Mapping World Cup 2021](https://osu.ppy.sh/community/forums/topics/1350409) will have the best mappers from all over the globe gather around and have an intense competition that will determine the next osu!catch Mapping Champions!

> Needless to say, such a contest is not going to be without prizes but all the details on that matter are currently work-in-progress. To keep track of the updates and learn more about the contest, we highly suggest you to check the [forum thread](https://osu.ppy.sh/community/forums/topics/1350409), and dive right into the [o!CMWC Discord server](https://discord.gg/eTUFAjf "Discord").

Reviewer: pretty sure this does benefit from a comma

```suggestion
Needless to say, such a contest is not going to be without prizes but, all the details on that matter are currently work-in-progress. To keep track of the updates and learn more about the contest, we highly suggest you to check the [forum thread](https://osu.ppy.sh/community/forums/topics/1350409), and dive right into the [o!CMWC Discord server](https://discord.gg/eTUFAjf "Discord").
```

> Alright, osu!catch folks! Hopefully, we've made your day with these two majestic contests, but make sure to compete responsibly, you wouldn't want to... *overdose* on fruit juice after all, would you?

Reviewer: ```suggestion
Alright, osu!catch folks! Hopefully we've made your day with these two majestic contests. However, make sure to compete responsibly, since you wouldn't want to... *overdose* on fruit juice after all, would you?
```

Reviewer 2: Pretty sure that comma is necessary.

There is no need to separate the sentence and start with "however," given that the tone should be informal. No need to change anything here.

Reviewer: I suggested to split the sentence because there were many commas, but alright. The first comma isn't necessary but can be there.

The "since" prevents comma splice, though.

Author: Added that "since" but otherwise it reads completely fine to me.

## Around the Community

Here's what's been going down in the community lately:

### Guide to making your first osu!mania 10K beatmap

Not so long ago, osu!mania got a new key-mode. After a long wait, 10K beatmaps can finally be ranked, and that's exactly what people need right now. But where does one start in mapping 10K? The same question came up on the 10K+ Rhythm Game Community server and so they gathered and created their own guide!

The 10K community representative [\[Sick\]Catcat](https://osu.ppy.sh/users/11544934) has posted their [10K osu!mania Mapping Guide](https://osu.ppy.sh/community/forums/topics/1336072) in the osu!mania subforum with everything you need to know to start mapping 10K osu!mania, covering anything from various pattern structures to useful tips and tricks in Easy to Hard difficulties.

> If this has piqued your interest, slap your favourite song into the editor and start mapping - everybody knows that 10K osu!mania is in urgent need of more ranked beatmaps, after all. And should you need more help, the [10K+ Rhythm Game Community](https://discord.gg/PwzcUzk) would be more than happy to assist.

Reviewer: ```suggestion
If this has piqued your interest, slap your favourite song into the editor and start mapping — everybody knows that 10K osu!mania is in urgent need of more ranked beatmaps, after all. And should you need more help, the [10K+ Rhythm Game Community](https://discord.gg/PwzcUzk) would be more than happy to assist.
```
